lte/gateway/c/oai/tasks/amf/amf_recv.cpp

```diff
@@ -118,6 +118,7 @@ int amf_handle_registration_request(
     OAILOG_DEBUG(LOG_NAS_AMF, "Processing REGITRATION_REQUEST message\n");
   }
   amf_registration_request_ies_t* params = new (amf_registration_request_ies_t);
+  memset(params, 0, sizeof(amf_registration_request_ies_t));
```

Comment:
This declaration should zero initialize: `amf_registration_request_ies_t* params = new amf_registration_request_ies_t();`